physics/dynamic_frame.hpp

```diff
@@ -59,6 +59,24 @@ class DynamicFrame {
       Instant const& t,
       DegreesOfFreedom<ThisFrame> const& degrees_of_freedom) const;
 
+  // The acceleration of a particle at rest in |ThisFrame| at the given
+  // |position| owing to non-inertial motion of |ThisFrame| and gravity,
+  // excluding components with a rotation.
+  // Let r be the radial vector (from the origin of |ThisFrame|) corresponding
+  // to |position|.
+  // Let r ↦ r″ be the vector field  of free fall accelerations from a rest in
+  // |ThisFrame| at t. This function returns
+  //   a = r″ - (rot r″) r / 2.
+  // In a rotating reference frame, this may equivalently be expressed using
+  // the second derivative of position with respect to the parametrization on
+  // the angle θ rather than time, which eliminates the Euler acceleration:
+  //   a = θ′² d²r/dθ², starting from a rest defined as dr/dθ = 0.
```

Comment:
Can we have TEX documentation for this change of variable?  (And for the `rot` formula above.)

Reply:
Not today.

Reply:
Not today. #3366.